modules/docs/sc_param.md

```diff
@@ -36,76 +36,76 @@ The sc_param module provides methods to help you handle parameters for your stre
+| log_level                               | number | 1                                                                             | the verbosity level for the logs. 1 = error + notice, 2 = error + warning + notice, 3 = error + warning + notice + debug (you should avoid using level 3)                                                                                                                                                                                     | any                                                                                                                                                                                                                                                                                      |                                                                                                                                                                                                                                                                                                  |
+| log_curl_commands                       | number | 0                                                                             | log ready to use curl commands when enabled (0 = disabled, 1 = enabled)                                                                                                                                                                                                                                                                       | any                                                                                                                                                                                                                                                                                      |                                                                                                                                                                                                                                                                                                  |
```

Comment:
?

Reply:
if you have a metric name that is

my$metric€name£  if those special characters are forbidden they are going to be replaced by the value of the metric_replacement_character

Reply:
I just don't understand the sentence "log ready to use curl commands when enabled"

Reply:
I was on the wrong problem :D 

this one parameter will print a curl command in your log file. Since you can just copy and paste this command in your shell, it is ready to use